docs/related.md

```diff
@@ -7,7 +7,7 @@ Standardised way of interchanging data between libraries, see
 
 ## DataFrame API Standard
 
-Project which aims to "provide a standard interface that encapsulates implementation details of dataframe libraries. This will allow users and third-party libraries to write code that interacts and operates with a standard dataframe, and not with specific implementations.", see [here](https://data-apis.org/dataframe-api/draft/).
+Now-discontinued project which aimed to "provide a standard interface that encapsulates implementation details of dataframe libraries. This will allow users and third-party libraries to write code that interacts and operates with a standard dataframe, and not with specific implementations.", see [here](https://data-apis.org/dataframe-api/draft/).
```

Comment:
> Now-discontinued project

Just out of curiosity: is that official?

Reply:
that's what they've told me